README.md

```diff
@@ -1,9 +1,30 @@
 # Awesome GitHub profiles
 
+See it here: [Awesome Github Profiles Website](https://eddiehubcommunity.github.io/awesome-github-profiles/)
```

Comment:
I think this can be removed and should be better added to the repository description. Only someone with maintainer permissions and above can do it. Maybe @eddiejaoude can help? 🙂

Reply:
Yes, that seems good.